app/models/comment.rb

```diff
@@ -1,4 +1,10 @@
 class Comment < ApplicationRecord
-  belongs_to :user
-  belongs_to :post
+  belongs_to :author, foreign_key: :author_id, class_name: 'User'
+  belongs_to :post, foreign_key: :post_id, class_name: 'Post'
+
+  after_create :update_comments_counter
+
+  def update_comments_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
**[OPTIONAL]**

- [ ] Kindly make this method private because this method should not be accessed by the users directly 👍 😊